openwisp_controller/config/static/config/js/utils.js

```diff
@@ -79,3 +79,29 @@ cleanData = function (data) {
         return data;
     }
 };
+
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
the whole codebase is using jQuery, so I think we should stick to this

Reply:
When I am using `$(document).ready()` checks are giving me an error that $ is not defined. What should I do about it? Also, it seems that utils.js is written in JavaScript. In many places, it is using `querySelector` and `document.getelement...`.  Please can you check and please correct me if I am in the wrong way.

Reply:
it's because of `noconflict`, use `jQuery`, check other files for reference.

Yes you're right that there are other `document.getelement`, it must have slipped my attention. but if you look at other similar JS files in this and other openwisp modules that are loaded in the django admin they are all using jQuery so it's good to keep consistency.

Reply:
Thanks, Actually I was going to use jQuery at the first time but because this file was using JavaScript so I preferred to use it. No worries I will address the changes by today but I think we should create an issue for it to change JavaScript to jQuery to maintain consistency.